internal/pkg/levee/propagation/propagation.go

```diff
@@ -199,6 +199,12 @@ func (prop *Propagation) visit(n ssa.Node, maxInstrReached map[*ssa.BasicBlock]i
 	case *ssa.Send:
 		prop.dfs(t.Chan.(ssa.Node), maxInstrReached, lastBlockVisited, false)
 
+	case *ssa.Slice:
+		prop.visitReferrers(n, maxInstrReached, lastBlockVisited)
+		// This allows taint to propagate backwards into the sliced value
+		// when the resulting slice is tainted
```

Comment:
Nit: the underlying collection is not necessarily a `Slice`.  Consider a more generic phrasing here.